ludwig/schema/model_types/llm.py

```diff
@@ -28,25 +29,22 @@ class LLMModelConfig(ModelConfig):
     """Parameters for LLM Model Type."""
 
     def __post_init__(self):
-        super().__post_init__()
-
-        if not self.model_name:
+        if self.base_model is None:
             raise ConfigValidationError(
-                "LLM requires `model_name` to be set. This can be any pretrained CausalLM on huggingface. "
+                "LLM requires `base_model` to be set. This can be a preset or any pretrained CausalLM on huggingface. "
                 "See: https://huggingface.co/models?pipeline_tag=text-generation&sort=downloads"
             )
 
+        super().__post_init__()
+
     model_type: str = schema_utils.ProtectedString("llm")
 
-    model_name: str = schema_utils.String(
-        default="",
-        allow_none=False,
+    base_model: BaseModelConfig = BaseModelDataclassField(
+        default=None,
         description=(
-            "The name of the model to use. This can be a local path or a "
-            "remote path. If it is a remote path, it must be a valid HuggingFace "
-            "model name. If it is a local path, it must be a valid HuggingFace "
-            "model name or a path to a local directory containing a valid "
-            "HuggingFace model."
+            "Base pretrained model to use. This can be one of the presets defined by Ludwig, a fully qualified "
+            "name of a pretrained model from the HuggingFace Hub, or a path to a directory containing a "
+            "pretrained model."
```

Comment:
Since this description shows up in the UI, I feel like it's a little confusing as written. Am curious why we need to say this is a "defined by Ludwig". Also should we tell the user they can choose "None" if they'd like to provide the path/name of a model that we don't have.

For example:
<img width="1152" alt="image" src="https://github.com/ludwig-ai/ludwig/assets/1681490/5985ee69-4ef5-4f1b-95ca-5feb6399d2c7">

Reply:
@arnavgarg1 @tgaddair thoughts here?

Reply:
I think this description makes sense for Ludwig users, but I guess it's a bit confusing for Predibase users. 
On the Predibase side, what we care about is `This can be one of the presets or a fully qualified name of a pretrained model from the HuggingFace Hub`.

What do we think?